jpo-ode-plugins/src/test/java/us/dot/its/jpo/ode/plugin/j2735/builders/TravelerMessageFromHumanToAsnConverterTest.java

```diff
@@ -482,4 +449,224 @@ void ensureComplianceWithJ2735Revision2024_containsMultipleOldFields() {
     });
   }
 
-}
+  @Test
+  void transformNodeLL_LLTypeSpecified() throws JsonUtilsException {
+    // prepare
+    ObjectNode node = JsonNodeFactory.instance.objectNode();
+    node.put("nodeLong", "0.0031024");
+    node.put("nodeLat", "0.0014506");
+    node.put("delta", "node-LL3");
+
+    // execute
+    ObjectNode result = TravelerMessageFromHumanToAsnConverter.transformNodeLL(node);
+
+    // verify
+    String expectedJson = "{\"delta\":{\"node-LL3\":{\"lat\":14506,\"lon\":31024}}}";
+    ObjectNode expected = JsonUtils.toObjectNode(expectedJson);
+
+    Assertions.assertEquals(expected.toString(), result.toString());
+  }
+
+  @Test
+  void transformNodeLL_LLTypeNotSpecified() throws JsonUtilsException {
+    // prepare
+    ObjectNode node = JsonNodeFactory.instance.objectNode();
+    node.put("nodeLong", "-0.0008192");
+    node.put("nodeLat", "-0.0013123");
+    node.put("delta", "node-LL");
```

Comment:
Note: The ODE allows the LL type to not be specified, which results in calling `determineNodeOffsetPointLLType`.